lib/data/landing_page_content_items/landing_page.yaml

```diff
@@ -1,4 +1,15 @@
 blocks:
+- type: main_navigation
+  links:
+  - label: Lorem for Ipsum
+    link: /ipsum-1
```

Comment:
Nitpick: obviously this is placeholder data, but I think `link` isn't needed for the active item?